### PR TITLE
create a list without bullet points style for the navbar

### PR DESCRIPTION
**Pull request by maxirozay**
So I'd like to have your feedback on what I did. I made it simple to understand if I'm doing right and if it's good I'll do more commits for this PR.

### CODE REVIEW

File: _includes/header.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
   </head>
   <body>
     <nav>
-      <ul>
+      <ul class="o-simplelist">
```
Comment (rspt) [inline _includes/header.html]: This isn't the best name out there.

I'll prefer `o-list` or `o-simple-list`.

File: _sass/objects.simplelist.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.o-simplelist {
+  list-style-type: none;
```
Comment (rspt) [inline _sass/objects.simplelist.scss]: That's good. One question though: is ot by design that you haven't changed the left margin (maybe it's padding, I don't remember by heart) of the list? Usually when we create a list object, we get rid of the list style type and the margin.

File: css/master.scss [hunk 3]
```diff
@@ -18,6 +18,7 @@
  *
  * OBJECTS
  * Grid.................Grid elements (using Flexbox)
+ * Simplelist.................Grid elements (using Flexbox)
```
Comment (rspt) [inline css/master.scss]: I'll name this `List` instead and put all list-related classes in there.

Also note that the `.` are only for aligning definition and provide visual clue and can change. The important thing is to align the definitions.

File: css/master.scss [hunk 4]
```diff
@@ -29,3 +30,4 @@
 @import "base.typography";
 
 @import "objects.grid";
+@import "objects.simplelist";
```
Comment (rspt) [inline css/master.scss]: Again, I'd name this simply `list`

File: _sass/objects.simplelist.scss (hunk 2 above)
Comment (maxirozay) [inline _sass/objects.simplelist.scss]: Yes I was looking at the w3c example and I just chose to keep the margin for now (i may change with an inline list).

Comment (maxirozay) [inline _sass/objects.simplelist.scss]: Do you think I should already remove the margin?

Comment (rspt) [inline _sass/objects.simplelist.scss]: If your idea is to create a basic list, like the following one then yes, you should remove the margin!:
```
list1<br>
list2<br>
list3<br>
list4<br>
```

That's why for me it makes no sense to remove the bullets without resetting the margin too.

**Comment by maxirozay** [comment]
I will make a better PR.

**Pull request closed without merging**